tests.yml

```diff
@@ -0,0 +1,66 @@
+# Run python-versions on {{ taskotron_item }} NVR build
+
+- hosts: localhost
+  remote_user: root
+  vars:
+    testcase: dist.python-versions
+    taskotron_generic_task: true
+    # below are fallback vars for local testing
+    artifacts: ./artifacts
+    taskotron_item: python-gear-0.11.0-1.fc27  # you should really override at least this :)
```

Comment:
What does this do? What is `python-gear` and why does it say we need to change it?

Reply:
That's a default variable value (a random recent koji package with python code in it), so that you can run `ansible-playbook tests.yml` without studying and adding any variables on the command line, and still see it doing _something_.

Reply:
ok